service/create/service.go

```diff
@@ -232,21 +232,36 @@ func (s *Service) Boot() {
 					}
 
 					// Run masters
-					if err := s.runMachines(runMachinesInput{
+					masterIDs, err := s.runMachines(runMachinesInput{
 						clients:             clients,
 						spec:                cluster.Spec,
 						tlsAssets:           tlsAssets,
 						clusterName:         cluster.Name,
 						keyPairName:         keyPairName,
 						instanceProfileName: instanceProfileName,
 						prefix:              prefixMaster,
-					}); err != nil {
+					})
+
+					if err != nil {
+						s.logger.Log("error", microerror.MaskAny(err))
+						return
+					}
+
+					masterIP := masterIDs[0]
```

Comment:
Will there always be at least one master? I would like to be more defensive and have some proper error handling for this.